client/common/directives/ModelFieldWYSIWYG/ModelFieldWYSIWYG.js

```diff
@@ -22,21 +22,23 @@ angular.module('dashboard.directives.ModelFieldWYSIWYG', [
 })
 
 .directive('modelFieldWysiwygEdit', function($compile, $cookies, $timeout, $modal, Config, FileUploadService) {
-  function getTemplate() {
-    var template = '\
+  function getTemplate(scope) {
+      let fontSizeArr = [7, 10, 12, 14, 16, 18, 24];
+      let minSize = scope.options.fontSizeRange != undefined ? scope.options.fontSizeRange.min : 1;
+      let maxSize = scope.options.fontSizeRange != undefined ? scope.options.fontSizeRange.max : 7;
```

Comment:
@leolynn Add range validation for the passed in values (shouldn't be less than 1, shouldn't be greater than the length of fontSizeArr, max should be greater than min)